typeDefs.js

```diff
@@ -288,6 +288,8 @@ const typeDefs = gql`
     updateUserByToken(user: UserInput!): User
     confirmUser(token: String!): Boolean
     recoverUser(email: String!, language: Int!): Boolean
+    changePassword(password: String!, token: String!): Boolean
```

Comment:
Please, rename it to the `resetPassword` to not match with the mutation `changePassword` that you can do from the user page

Reply:
changed